### PR TITLE
Add Tesseract OCR for preprocessor 

### PR DESCRIPTION
**Pull request by lolipopshock**
```bash
pawls preprocess ocr path-to-pdf
```

### CODE REVIEW

File: cli/pawls/preprocessors/tesseract.py [hunk 1]
```diff
@@ -0,0 +1,104 @@
+from typing import List, Tuple, Dict
+import csv
+import io
+
+import pytesseract
+import pandas as pd
+from pdf2image import convert_from_path
+from PIL import Image
```
Comment (DeNeutoy) [inline cli/pawls/preprocessors/tesseract.py:8]: If this is only for the type, I suggest dropping it - if not, can you add it to requirements.txt?

**Review by DeNeutoy — APPROVED** [review]
LGTM, one small comment!

File: cli/pawls/preprocessors/tesseract.py (hunk 1 above)
Comment (lolipopshock) [reply cli/pawls/preprocessors/tesseract.py:8]: Thanks! Just for type - OK I'll remove it!

**Comment by naourass** [comment]
Hello, I'm planning to try PAWLS with Arabic PDF scanned documents.  Is there anything I should change to make tesseract use Arabic (or another language) as a target language ?

**Pull request merged**